test/test_install.py

```diff
@@ -142,7 +142,7 @@ def test_install_miniconda_datalad(tmp_path):
     assert (miniconda_path / bin_path("datalad")).exists()
 
 
-@pytest.mark.miniconda
+@pytest.mark.unforgeable
```

Comment:
what does 'unforgeable' means?  also `not unforgeable` double negation requires mental bool algebra and for me it is especially difficult since I neither know what 'forgeable' is

```
$> dict unforgeable
No definitions found for "unforgeable"
$> dict forgeable 
No definitions found for "forgeable"
```

and if it means "needing sudo" -- I don't think it is not needed for this particular installation since `miniconda` is installable into user space without sudo.

Reply:
@yarikoptic "forgeable" here means "suitable for conda-forge".

Reply:
@yarikoptic There are two types of "unforgeable" tests here:

* The ones previously marked with "miniconda" install Miniconda, which doesn't work when you're already inside a Conda environment.  The conda-forge tests run inside Conda, so these tests fail on conda-forge.
* Two recently-added tests for the rclone and git-annex-remote-rclone components try to install the programs in question "globally", i.e., inside `/usr/local/bin`, and if they get `PermissionDenied`, they try again with `sudo mv`.  In the environments that conda-forge tests run in, the tests ends up trying to invoke `sudo`, but it requires a password in these environments, so the tests fail.

Reply:
oh, so we need to mark'ers and I would prefer giving them a bit less cryptic names:
- `unforgeable` doesn't have anything to do with "conda-forge" and rather `installs_conda`. So testing on any "conda" distribution/environment should not be ran, not only on conda-forge,
- is `needs_sudo`, so e.g. any user could run tests but skip those needing sudo since he/she wouldn't have it.